cmd/lock-rpc-server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"math/rand"
 	"net/rpc"
-	"path"
+	pathutil "path"
```

Comment:
I do not see any `path` variable currently.  Even in future, why someone will bring generic `path` variable name to conflict with `path` package.